lib/libmbedtls/core/rsa.c

```diff
@@ -627,7 +627,7 @@ TEE_Result crypto_acipher_rsassa_verify(uint32_t algo,
 
 	bigint_size = crypto_bignum_num_bytes(key->n);
 	if (sig_len < bigint_size) {
-		res = TEE_ERROR_MAC_INVALID;
+		res = TEE_ERROR_SIGNATURE_INVALID;
```

Comment:
+1.
From the GP spec (v1.0 to v1.3), this error code is applicable to `TEE_AsymmetricVerifyDigest()` while `TEE_ERROR_MAC_INVALID` is applicable to `TEE_MACCompareFinal()` and  `TEE_AEDecryptFinal()`.